uiowa/uiowa
# Add Easy Breadcrumb to Periodical Split

This PR introduces the easy_breadcrumb module to the periodical split (e.g. iwp91stmeridian) to provide faux breadcrumbs for non-menu based URL patterns.

Resolves https://github.com/uiowa/uiowa/issues/6013

# To Test

```
ddev composer install && ddev blt ds --site 91stmeridian.iwp.uiowa.edu && ddev drush @iwp91stmeridian.local pathauto:aliases-generate update canonical_entities:node && ddev drush @iwp91stmeridian.local cr && ddev drush @iwp91stmeridian.local uli issues/volume-6-issue-1-61-spring-2008/last-bird-burial-master
```

See that it has a breadcrumb trail based on the URL pattern where `issues` is unlinked and the `volume 6...` is linked.

Go to https://iwp91stmeridian.uiowa.ddev.site/node/46/edit
Provide a menu link that is nested under the about page. Save.
View the page and see that the path based on menu is as expected.

Go back and edit, override the generated URL alias with a complex fake nested path. Save.
View the page and see that it still respects the menu settings for the breadcrumb.


# Deployment Instructions

run `drush pathauto:aliases-generate update canonical_entities:node` on 91stmeridian that uses the feature
